keyboards/keychron/k10_pro/iso/rgb/info.json

```diff
@@ -1,133 +1,32 @@
-                {"matrix":[3,10], "x":10.75, "y":3.25},
-                {"matrix":[3,11], "x":11.75, "y":3.25},
-                {"matrix":[3,13], "x":12.75, "y":3.25},
-                {"matrix":[2,13], "x":13.75, "y":2.25, "w":1.25, "h":2},
-                {"matrix":[3,17], "x":18.5, "y":3.25},
-                {"matrix":[3,18], "x":19.5, "y":3.25},
-                {"matrix":[3,19], "x":20.5, "y":3.25},
-
-                {"matrix":[4,0], "x":0, "y":4.25, "w":1.25},
-                {"matrix":[4,1], "x":1.25, "y":4.25},
-                {"matrix":[4,2], "x":2.25, "y":4.25},
-                {"matrix":[4,3], "x":3.25, "y":4.25},
-                {"matrix":[4,4], "x":4.25, "y":4.25},
-                {"matrix":[4,5], "x":5.25, "y":4.25},
-                {"matrix":[4,6], "x":6.25, "y":4.25},
-                {"matrix":[4,7], "x":7.25, "y":4.25},
-                {"matrix":[4,8], "x":8.25, "y":4.25},
-                {"matrix":[4,9], "x":9.25, "y":4.25},
-                {"matrix":[4,10], "x":10.25, "y":4.25},
-                {"matrix":[4,11], "x":11.25, "y":4.25},
-                {"matrix":[4,13], "x":12.25, "y":4.25, "w":2.75},
-                {"matrix":[4,15], "x":16.25, "y":4.25},
-                {"matrix":[4,17], "x":18.5, "y":4.25},
-                {"matrix":[4,18], "x":19.5, "y":4.25},
-                {"matrix":[4,19], "x":20.5, "y":4.25},
-                {"matrix":[4,20], "x":21.5, "y":4.25, "h":2},
-
-                {"matrix":[5,0], "x":0, "y":5.25, "w":1.25},
-                {"matrix":[5,1], "x":1.25, "y":5.25, "w":1.25},
-                {"matrix":[5,2], "x":2.5, "y":5.25, "w":1.25},
-                {"matrix":[5,6], "x":3.75, "y":5.25, "w":6.25},
-                {"matrix":[5,10], "x":10, "y":5.25, "w":1.25},
-                {"matrix":[5,11], "x":11.25, "y":5.25, "w":1.25},
-                {"matrix":[5,12], "x":12.5, "y":5.25, "w":1.25},
-                {"matrix":[5,13], "x":13.75, "y":5.25, "w":1.25},
-                {"matrix":[5,14], "x":15.25, "y":5.25},
-                {"matrix":[5,15], "x":16.25, "y":5.25},
-                {"matrix":[5,16], "x":17.25, "y":5.25},
-                {"matrix":[5,17], "x":18.5, "y":5.25, "w":2},
-                {"matrix":[5,19], "x":20.5, "y":5.25}
-            ]
+    "features": {
+        "led_matrix": true
+    },
+    "led_matrix": {
+        "driver": "CKLED2001",
+        "animations": {
+            "none": true,
+            "solid": true,
+            "breathing": true,
+            "band_pinwheel": true,
+            "band_spiral": true,
+            "cycle_left_right": true,
+            "cycle_up_down": true,
+            "cycle_out_in": true,
+            "dual_beacon": true,
+            "solid_reactive_simple": true,
+            "solid_reactive_multiwide": true,
+            "solid_reactive_multinexus": true,
+            "solid_splash": true,
+            "wave_left_right": true,
+            "wave_up_down": true,
+            "effect_max": true
```

Comment:
we ned rgb matrix here

Reply:
Seems like I overlooked this mistake here. I'll fix it up and check to make sure other boards aren't affected by this.

Reply:
Resolved in https://github.com/Keychron/qmk_firmware/commit/71515b61904d8d31fdfdf4b0075759d6cd420b65.